app/experimenter/visualization/api/v1/views.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+from django.core.files.storage import default_storage
+from django.http import HttpResponse
+from django.views import View
```

Comment:
Oh!  Since this is an API you could use the [Django Rest Framework APIView](https://www.django-rest-framework.org/api-guide/views/) as your base here, that way it handles JSON automatically and any global configuration we add applies to it as well.  And it might even get automatically documented somehow?

Reply:
Oh actually using using `[RetrieveAPIView](https://www.django-rest-framework.org/api-guide/generic-views/#retrieveapiview)` should do exactly what you want

Reply:
If using a class based view doesn't work well you can also keep it in the form it's in now and just decorate it to be a [DRF function based view](https://www.django-rest-framework.org/api-guide/views/#function-based-views)